{"name": "test_div[小数除于字符串异常用例]", "status": "passed", "parameters": [{"name": "a", "value": "22.0"}, {"name": "b", "value": "'11'"}, {"name": "expect", "value": "'TypeError'"}], "start": 1603368605853, "stop": 1603368605853, "uuid": "13b934a9-fff0-4a51-8479-bad129e51ae3", "historyId": "234ddac65658cee96db4496411b76139", "testCaseId": "1a86939ae8c138566bc6b8885470dbad", "fullName": "test_calc1.TestCalcTaskOne#test_div", "labels": [{"name": "tag", "value": "div"}, {"name": "tag", "value": "run(order=2)"}, {"name": "suite", "value": "test_calc1"}, {"name": "subSuite", "value": "TestCalcTaskOne"}, {"name": "host", "value": "lingz-mac.lan"}, {"name": "thread", "value": "50460-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc1"}]}